set_kinova_in_ulna.py

```diff
@@ -0,0 +1,67 @@
+import bioviz
+import numpy as np
+import biorbd
+
+# decimal option
+np.set_printoptions(precision=10)
+
+# model_name = "KINOVA_arm_deprecated.bioMod"
+model_name = "models/KINOVA_merge_6dof.bioMod"
+
+biorbd_model = biorbd.Model(model_name)
+
+# Now the model is loaded as a biorbd object
+
+# Q = np.ones(biorbd_model.nbQ())
+Q = np.array(
+    [
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        0.0,
+        -0.0398,
+        -0.0752,
+        0.0291,
+        2.8037,
+        -1.6032,
+        -0.1205,
+    ]
+)
+# Q is the vector that we get after manually placed the support into the simulation
+
+print(biorbd_model.nbQ())
+
+# In our configuration, ulna is segment 30 and support is segment 40 ( it can change if KINOVA_merge.bioMod is
+# modify) So, we want to find the rototrans ( aka homogeneous transform) of ulna and the support form local to global
```

Comment:
Please, replace "modify" with "modified".
And the extra space in the parenthesis "( aka homogeneous transform)"

Reply:
"form" with "from"